### PR TITLE
fix(control-tower): remove feature flag since Control Tower is a mandatory feature

### PR DESCRIPTION
**Pull request by Reb-B**
References to control tower as an optional parameter are removed from superwerker.template.yaml

### CODE REVIEW

**Comment by sbstjn** [comment]
I started the testing pipeline to check if this works as intended :)

**Comment by sbstjn** [comment]
pipeline was fine 🎉

**Review by s0enke — APPROVED** [review]

File: templates/superwerker.template.yaml [hunk 1]
```diff
@@ -151,19 +141,10 @@ Conditions:
   UsingDefaultBucket: !Equals [!Ref QSS3BucketName, 'aws-quickstart']
   IncludeNotifications: !Not [!Equals [ !Ref NotificationsMail, '' ]]
   IncludeBudget: !Equals [ !Ref IncludeBudget, 'Yes' ]
-  IncludeControlTower: !Equals [ !Ref IncludeControlTower, 'Yes' ]
-  IncludeGuardDuty: !And
```
Comment (s0enke) [inline templates/superwerker.template.yaml:155]: @Reb-B it might make sense to split out the reformatting into another commit / PR, so that this commit/PR is scoped to "remove feature flag since Control Tower is a mandatory feature". What do you think?

**Review by s0enke — CHANGES_REQUESTED** [review]

File: templates/superwerker.template.yaml (hunk 1 above)
Comment (s0enke) [reply templates/superwerker.template.yaml:155]: Forget my comment, I was blind :)

**Review by s0enke — APPROVED** [review]

**Comment by s0enke** [comment]
I reworded the commit to the same as the PR title, now it's good to go.

Thanks @Reb-B!
